### DISCUSSION TITLE
Logback custom appender to push logs to ElasticSearch via Open Telemetry

### DISCUSSION THREAD

**Discussion by hegdemahendra** [open-telemetry/community#889] [Community Discussions]
Hello Team,

I am using apache Nifi , which uses logback framework for logging.
I have written a custom log appender to push logs to MongoDB.
Now I have a requirement to push logs in specific format (json) to ElasticSearch.

As few other apps are already using open telemetry to push data to ES, i have been asked to do so.

So now I am looking for right way to push data from my custom log appender (java class) to Open Telemetry.

Could some please point me to the usefull example which I can refer ?

Thanks
Mahendra

**Comment by trask** [comment]
hi @hegdemahendra! can you ask this in https://github.com/open-telemetry/opentelemetry-java? that will get the right 👀 on your question 👍

**Reply by hegdemahendra** [reply]
Thanks for pointing to the right forum @trask
